docs/src/dedicated-gen-2/overview.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Platform.sh Dedicated Generation 2"
+weight: 1
+sidebarTitle: "Overview"
+layout: single
+---
+
+Platform.sh Dedicated Generation 2 provides a scalable solution as an additional option on top of your existing Grid applications.
+It provides redundant configuration with a minimum of three Virtual Machine instances. Every service is replicated across all three virtual machines in a failover configuration (as opposed to sharding), allowing a site to remain up even if one of the VMs is lost entirely.
+Dedicated Generation 2 works nearly identically to Grid environments, and does not require additional configuration on your part. The only difference is that of service availability.
+
+## Why a Generation 2?
+
+Even though our existing [dedicated infrastructure](/dedicated/overview) ensures better uptime and availability for your applications and services, configuration or application topology changes have to go through our Custiomer's Success team, as the VMs are provisionned by us. Our teams have been working hard to bring you a solution that gives you both the high availability from our Dedicated offer and the self-service flexibility and features of the Grid: Dedicated Generation 2. This means that you will now be able to edit your configuration yourself – as you would do on the Grid in your `.platform/services.yaml` and `.platform.app.yaml` files – and see those changes in your Dedicated Generation 2 environments when pushed without having to open a ticket.
+
+## Features
+
+* Domains can be added to your development environments, such as to a dedicated staging environment.
+* Three-Virtual Machine for your master branch.
+* Backups on Production do not freeze in the process.
+* You can deploy your application in any of our [supported cloud providers](/development/faq.md#which-geographic-zones-does-platformsh-cover).
```

Comment:
Kept this from Otavio's version. Not sure it makes sense to keep it? Or should be re-worked at least?

Reply:
Yeah. Not Dedicated-Gen-2-specific so not sure it's even relevant to keep this here? idk
I feel like we could get rid of this section entirely...?

Reply:
> * Backups on Production do not freeze in the process.

yeah that's not true. dg3 backups are the same as the grid.